Assignment-4/app/src/Components/SidebarNavigation.js

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from "react";
+import Sidebar from "../Components/Sidebar";
+
+const SidebarNavigation = () => {
+  const [isSidebarOpen, setSidebarOpen] = useState(false);
+
+  const toggleSidebar = () => {
+    setSidebarOpen(!isSidebarOpen);
+  };
+
+  return (
+    <div>
+      <header>
+        <button onClick={toggleSidebar}>
+          <img
```

Comment:
![image](https://github.com/kaushalyad321/ReactJs/assets/83696224/6a275f02-aa18-49b5-a0a9-6de7342bf786)
All these links navigating to home page only